File: 0075-sort-colors/0075-sort-colors.py
class Solution:
    def sortColors(self, nums: List[int]) -> None:
        red = white = blue = 0
        ans = []
        for i in nums:
            if i==0:
                red+=1
            elif i ==1:
                white+=1
            else:
                blue+=1
        for i in range(len(nums)):
            if red:
                nums[i] = 0
                red-=1
            elif white:
                nums[i] = 1
                white-=1
            else:
                nums[i] = 2
                blue -=1
                
            
        
        

File: 0075-sort-colors/0075-sort-colors.cpp
class Solution {
public:
    void sortColors(vector<int>& nums) {
        int zero = 0, one = 0, two = 0;
        for(int num : nums){
            if(num==0) zero++;
            else if(num == 1) one++;
            else two++;
        }
        int i = 0;
        while(zero){
            nums[i++] = 0;
            zero--;
        }
        while(one){
            nums[i++] = 1;
            one--;
        }
        while(two){
            nums[i++] = 2;
            two--;
        }
    }
};